.github/workflows/aznfs-test.yml

```diff
@@ -0,0 +1,354 @@
+name: Test AZNFS
+
+on:
+  workflow_dispatch:
+    inputs:
+      versionName:
+        description: 'Version Name'
+        required: true
+      storageAccounts:
+        description: 'Space-separated Storage Accounts'
+        required: true
+
```

Comment:
can we define an env block here and put all env variables needed by any step in that. f.e. I can see at least following coming over there

RELEASE_NUMBER
STORAGE_ACCOUNTS

having all env variables in one place is more readable